backend/src/sql/list-dreps.sql

```diff
@@ -230,13 +230,13 @@ GROUP BY
   off_chain_vote_drep_data.image_hash
 ORDER BY
   CASE
-    WHEN $2 = 'Random' THEN RANDOM()
-    WHEN $2 = 'VotingPower' THEN DRepDistr.amount
-    WHEN $2 = 'RegistrationDate' THEN EXTRACT(EPOCH FROM newestRegister.time)
+    WHEN $2 = 'VotingPower' THEN DRepDistr.amount::numeric
+    WHEN $2 = 'RegistrationDate' THEN EXTRACT(EPOCH FROM newestRegister.time)::numeric
```

Comment:
I think it should be rather the first registration instead of the latest registration or update transaction